### PR TITLE
feature: Adding ability to sort the import map alphabetically

### PR DESCRIPTION
**Pull request by TheMcMurder**
This is something I'd like just because it would be convenient and make some of the larger import maps easier to read.

I'm pretty sure this is all I would need to change to support this feature.

### CODE REVIEW

File: test/import-map.test.js [hunk 1]
```diff
@@ -4,22 +4,24 @@ const {
...
 
-beforeAll(() => {
-  setConfig({
-    manifestFormat: "importmap",
-    packagesViaTrailingSlashes: true,
-    locations: {
-      prod: "memory://prod",
-    },
+describe(`/import-map.json`, () => {
+  let errorSpy;
```
Comment (TheMcMurder) [inline test/import-map.test.js:8]: Added this errorSpy so the tests wouldn't actually error to the console and clutter up the output

Comment (TheMcMurder) [reply test/import-map.test.js:8]: I wanted to scope it as well so I moved the `beforeAll` and `beforeEach` blocks into the `describe` block

File: test/alphabetical-import-map.test.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+      .expect(200)
+      .expect("Content-Type", /json/);
+    return setupRequest.then((response) => {
+      expect(JSON.stringify(response.body)).toBe(
+        `{"imports":{"c":"/c-1.mjs","b":"/b-1.mjs"},"scopes":{}}`
+      );
+    });
+  });
+
+  it(`should place the import into the map alphabetically instead of just at the end`, async () => {
```
Comment (joeldenning) [inline test/alphabetical-import-map.test.js:98]: This test name was copied over but says the reverse of what it tests

**Review by joeldenning — COMMENTED** [review]
Looks good, just one small comment

**Review by joeldenning — APPROVED** [review]

**Comment by joeldenning** [comment]
Released in https://github.com/single-spa/import-map-deployer/releases/tag/v4.5.0

**Pull request merged**